darwin/datatypes.py

```diff
@@ -301,7 +394,7 @@ def make_bounding_box(
         AnnotationClass(class_name, "bounding_box"),
         {"x": round(x, 3), "y": round(y, 3), "w": round(w, 3), "h": round(h, 3)},
         subs or [],
-        slot_names=slot_names or [],
+        slot_names=slot_names or [] or [],
```

Comment:
```suggestion
        slot_names=slot_names or [],
```

Reply:
there are a few occurrences in this file, unless I'm missing something

Reply:
Fixed, bad grep/replace.